### PR TITLE
Many fixes

### PR DESCRIPTION
**Pull request by jfbastien**
* Move history to the end
* Fix stray newlines
* Overhaul both English and C++ wording

### CODE REVIEW

File: doc/proposal/expected/d0323r3.md [hunk 1]
```diff
@@ -969,247 +1377,282 @@ template <class T, class E>
...
+The constructor is explicit if and only if `is_convertible_v<U const&, T>` is
+false or `is_convertible_v<G const&, E> is false.
 
 ###########################################################################
 
 ```c++
-	template <class U, class G>
-	EXPLICIT expected(expected<U,G>&& rhs) noexcept(see below);
+    template <class U, class G>
+    explicit expected(expected<U,G>&& rhs) noexcept(see below);
```
Comment (viboes) [inline doc/proposal/expected/d0323r3.md]: THe intent was EXPLICIT in upper case as we have for pair. The meaning of EXPLICIT differs. See #1600.

Comment (jfbastien) [reply doc/proposal/expected/d0323r3.md]: Sounds good, I put it back.

File: doc/proposal/expected/d0323r3.md [hunk 2]
```diff
@@ -163,82 +165,266 @@ Using `expected<double, errc>`, the code becomes:
...
+int divide2(int i, int j)
 {
-	auto e = safe_divide(i,j);
-	if (!e && e.error().value() == arithmetic_errc::divide_by_zero) {
-		return 0;
-	}
-	return e;
+    auto e = safe_divide(i, j);
+    if (!e)
+        switch (e.error().value()) {
```
Comment (viboes) [inline doc/proposal/expected/d0323r3.md:387]: I would need to fix this `e.error().value()` and use directly `e.error()`. It was already there.

File: doc/proposal/expected/d0323r3.md [hunk 3]
```diff
@@ -163,82 +165,266 @@ Using `expected<double, errc>`, the code becomes:
...
+## URL parsing
+
+The following is how WebKit-based browsers [parse URLs] and use `std::expected`
+to denote failure:
+
+```c++
+template<typename CharacterType>
+std::expected<uint32_t, URLParser::IPv4PieceParsingError>
+URLParser::parseIPv4Piece(CodePointIterator<CharacterType>& iterator, bool& didSeeSyntaxViolation)
+{
```
Comment (viboes) [inline doc/proposal/expected/d0323r3.md:222]: Wondering if the implementation of this function adds value as what is important is on the use case that uses it.

**Comment by viboes** [comment]
I have decided to merge it as it is now, and fix the possible issues later.

File: doc/proposal/expected/d0323r3.md (hunk 3 above)
Comment (jfbastien) [reply doc/proposal/expected/d0323r3.md:222]: I'm worried that other examples will be dismissed as "toys". This is production code, deployed to a few million devices, with performance constraints. It's significantly harder to complain that the usecase isn't compelling with that type of code :)

**Pull request merged**